data/pg/ACCESSCONTROL-ROLES/roles.json

```diff
@@ -236,6 +236,714 @@
+    },
+    {
+      "code": "BPAREG_DOC_VERIFIER",
+      "name": "BPAREG doc verifier",
+      "description": "BPA Stakeholder workflow document verifier"
+    },
+    {
+      "code": "BPAREG_EMPLOYEE",
+      "name": "BPAREG Employee",
+      "description": "BPA Stakeholder Employee"
+    },
+    {
+      "code": "BPA_APPROVER",
+      "name": "BPA Services Approver",
+      "description": "BPA Services workflow Approver"
+    },
+    {
+      "code": "BPA_VERIFIER",
+      "name": "BPA Services verifier",
+      "description": "BPA Services workflow document & field inspection verifier"
+    },
+    {
+      "code": "CITY_ADMIN",
+      "name": "Admin of a ULB",
+      "description": "Admin of ULB to get access to the ULB level"
+    },
+    {
+      "code": "WS_CEMP",
+      "name": "WS Counter Employee",
+      "description": "WS counter employee can take action as citizen"
+    },
+    {
+      "code": "WS_DOC_VERIFIER",
+      "name": "WS Document Verifier",
+      "description": "WS Document Verifier"
+    },
+    {
+      "code": "WS_FIELD_INSPECTOR",
+      "name": "WS Field Inspector",
+      "description": "WS Inspector can take action like verify and forword, send back to do"
+    },
+    {
+      "code": "WS_APPROVER",
+      "name": "WS Approver",
+      "description": "WS Approver can approve the connection application"
+    },
+    {
+      "code": "WS_CLERK",
+      "name": "WS Clerk",
+      "description": "WS Clerk can activate the connection application"
+    },
+    {
+      "code": "SW_CEMP",
+      "name": "SW Counter Employee",
+      "description": "SW counter employee can take action as citizen"
+    },
+    {
+      "code": "SW_DOC_VERIFIER",
+      "name": "SW Document Verifier",
+      "description": "SW Document Verifier"
+    },
+    {
+      "code": "SW_FIELD_INSPECTOR",
+      "name": "SW Field Inspector",
+      "description": "SW Inspector can take action like verify and forword, send back to do"
+    },
+    {
+      "code": "SW_APPROVER",
+      "name": "SW Approver",
+      "description": "SW Approver can approve the connection application"
+    },
+    {
+      "code": "SW_CLERK",
+      "name": "SW Clerk",
+      "description": "SW Clerk can activate the connection application"
+    },
+    {
+      "code": "PT_CEMP",
+      "name": "PT Counter Employee",
+      "description": "Property Counter Employee"
+    },
+    {
+      "code": "PT_DOC_VERIFIER",
+      "name": "PT Doc Verifier",
+      "description": "Property Document Verifier"
+    },
+    {
+      "code": "PT_FIELD_INSPECTOR",
+      "name": "PT Field Inspector",
+      "description": "Property Field Inspector"
+    },
+    {
+      "code": "PT_APPROVER",
+      "name": "PT Counter Approver",
+      "description": "PT Approver"
+    },
+    {
+      "code": "SYSTEM",
+      "name": "System user",
+      "description": "System user role"
+    },
+    {
+      "code": "BPA_FIELD_INSPECTOR",
```

Comment:
add only FSM and Sanitation required roles, skip other mission related like BPA PT etc